code/modules/mob/living/living_defense.dm

```diff
@@ -200,7 +200,7 @@
 /mob/living/proc/resist_fire(datum/source)
 	SIGNAL_HANDLER
 	fire_stacks = max(fire_stacks - rand(3, 6), 0)
-	Paralyze(80)
+	Paralyze(30)
```

Comment:
```suggestion
	Paralyze(3 SECONDS)
```

Reply:
apparently this is the amount of paralyze whatever the fuck that means not the actual length of paralyze but its basically +20/10 length of the set number